### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by ghost** [custom-components/ble_monitor#1110] [Q&A]
I have everything installed but how do I acutually add the device and view its output. Do i need to have a bluetooth dongle or is just using a pi 3 okay

**Comment by Ernst79** [comment]
A pi 3 is ok, it has bluetooth build in. Sensors should be automatically added, but a restart might be needed for the first time

**Reply by ghost** [reply]
ive been messing with this for hours. I have the govee h5102 sensor and i cant for life of me get it to show up.

**Comment by Ernst79** [comment]
Govee needs active scan to be enabled in the ble monitor settings

**Reply by ghost** [reply]
ive got that checked on

**Comment by Ernst79** [comment]
Anything in the logs? If not, try to enable debug logging and enable the option to report unknown sensors (if you know the MAC address, only for that mac. Post the logs here and I’ll check it tomorrow.
